controllers/progressivesync_controller.go

```diff
@@ -60,171 +61,69 @@ type ProgressiveSyncReconciler struct {
 
 // Reconcile performs the reconciling for a single named ProgressiveSync object
 func (r *ProgressiveSyncReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	log := r.Log.WithValues("ProgressiveSync", req.NamespacedName)
+	log := r.Log.WithValues("progressivesync", req.NamespacedName)
```

Comment:
Why did we change the case here?

Reply:
To be consistent with https://github.com/Skyscanner/applicationset-progressive-sync/blob/main/controllers/progressivesync_controller.go#L72 where we say `applicationset`. I thought having logs in lower-case would it be better.